README.rst

```diff
@@ -361,6 +362,42 @@ Note that ``THREADS`` backend is not recommended since `Kokkos does not guarante
 
 .. installation_LKokkos-end-inclusion-marker-do-not-remove
 
+.. installation_LTensor-start-inclusion-marker-do-not-remove
+
+Lightning-Tensor installation
+*****************************
+Lightning-Tensor requires CUDA 12 and the `cuQuantum SDK <https://developer.nvidia.com/cuquantum-sdk>`_ (only the `cutensornet <https://docs.nvidia.com/cuda/cuquantum/latest/cutensornet/index.html>`_ library is required).
+The SDK may be installed within the Python environment ``site-packages`` directory using ``pip`` or ``conda`` or the SDK library path appended to the ``LD_LIBRARY_PATH`` environment variable.
+Please see the `cuQuantum SDK <https://developer.nvidia.com/cuquantum-sdk>`_ install guide for more information.
+
+Install Lightning-Tensor from source
+====================================
+
+Lightning-Qubit should be installed before Lightning-Tensor:
+
+.. code-block:: console
+
+    git clone https://github.com/PennyLaneAI/pennylane-lightning.git
+    cd pennylane-lightning
+    pip install -r requirements.txt
```

Comment:
`requirements-dev.txt` includes `cutensornet-cu12`
```suggestion
    pip install -r requirements-dev.txt
```

Reply:
yea, `cutensornet-cu12` is included in `requirements-dev.txt`.  But the installation of `lightning_qubit` doesn't require `cutensornet-cu12` or `custatevec-cu12` listed in `requirements-dev.txt`.